docs/advanced-selectors.md

```diff
@@ -0,0 +1,1357 @@
+# Advanced Selectors
+
+If you need a more complex way of selecting which spans of a trace will be asserted, you can write your selector as a query. This will enable you to express more complex conditions to select spans.
+
+The query language is based on `CSS`. We don't support all features of CSS, just a few features. If you think there is one that can help users to be more effective, open an issue and request it.
```

Comment:
I would remove the `based on CSS` part, since it might be confusing. The synthax is loosely based on css, but it's not really similar